tests/integration/test_data/gomod_with_deps/output.json

```diff
@@ -0,0 +1,261 @@
+        },
+        {
+          "type": "go-package",
+          "name": "os",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "path",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "reflect",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "rsc.io/quote",
+          "version": "v1.5.2"
+        },
+        {
+          "type": "go-package",
+          "name": "rsc.io/sampler",
+          "version": "v1.3.0"
+        },
+        {
+          "type": "go-package",
+          "name": "runtime",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "runtime/internal/atomic",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "runtime/internal/math",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "runtime/internal/sys",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "runtime/internal/syscall",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "sort",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "strconv",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "strings",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "sync",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "sync/atomic",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "syscall",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "time",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "unicode",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "unicode/utf8",
+          "version": null
+        },
+        {
+          "type": "go-package",
+          "name": "unsafe",
+          "version": null
+        }
+      ]
+    },
+    {
+      "type": "gomod",
+      "path": ".",
+      "name": "github.com/cachito-testing/cachito-gomod-with-deps",
+      "version": "v0.0.0-20201001160613-4c65d49cae6b",
+      "dependencies": [
+        {
+          "type": "gomod",
+          "name": "golang.org/x/text",
+          "version": "v0.0.0-20170915032832-14c0d48ead0c"
+        },
+        {
+          "type": "gomod",
+          "name": "rsc.io/quote",
+          "version": "v1.5.2"
+        },
+        {
+          "type": "gomod",
+          "name": "rsc.io/sampler",
+          "version": "v1.3.0"
+        }
+      ]
+    }
+  ],
+  "environment_variables": []
```

Comment:
Wait, why are `environment_variables` empty? They shouldn't be empty :thinking:

Reply:
Oh, I know why. I introduced a bug in #36, I'll fix it

Reply:
yeah I noticed this anomaly too yesterday, but I wanted to take it to separate MR with that Makefile (I hate rewriting git history each time here in bigger PR :) ). If I build and run test locally it has output.json has env-vars http://pastebin.test.redhat.com/1085236 if I use latest cachi2 from quay or the one from CI -> in that case they are missing